crime-applications-adaptor/src/test/java/uk/gov/justice/laa/crime/applications/adaptor/mapper/crimeapply/BenefitsMapperTest.java

```diff
@@ -0,0 +1,178 @@
+package uk.gov.justice.laa.crime.applications.adaptor.mapper.crimeapply;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import uk.gov.justice.laa.crime.applications.adaptor.enums.BenefitDetails;
+import uk.gov.justice.laa.crime.applications.adaptor.model.crimeapplicationsadaptor.common.AssessmentDetail;
+import uk.gov.justice.laa.crime.applications.adaptor.model.criminalapplicationsdatastore.general.Benefit;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class BenefitsMapperTest {
+
+    private BenefitsMapper benefitsMapper;
+    private final int AMOUNT = 100;
```

Comment:
I think these should be static also as the variable names are those of a constant.

Reply:
done